src/libraries/Microsoft.Extensions.Caching.Memory/src/CacheEntry.cs

```diff
@@ -318,10 +326,10 @@ private static void InvokeCallbacks(CacheEntry entry)
                 {
                     registration.EvictionCallback?.Invoke(entry.Key, entry.Value, entry.EvictionReason, registration.State);
                 }
-                catch (Exception)
+                catch (Exception e)
                 {
                     // This will be invoked on a background thread, don't let it throw.
-                    // TODO: LOG
+                    entry._logger.LogInformation(e, "EvictionCallback invoked failed");
```

Comment:
FYI, the PR that introduced the //TODO: LOG comment: https://github.com/aspnet/Caching/pull/4/files

Reply:
It only took 6 years for someone to notice 😁.

Reply:
This should probably be logged at the error level, it's application code that's throwing.

Reply:
@Tratcher Thank you and I replace the `LogInformation` to `LogError`